### ISSUE TITLE
Extension to mixed signal range?

### ISSUE THREAD

**Issue by tony1tf** [yytseng/gds2000tools#1]
I see you havn’t been active for 6 years, but GW Instek have mixed signal scopes with 16 digital and 4 analog. Have you done any work on drivers for these models? I have an MSO2204EA and would love to use it with PulseView. We have customised decoders for PulseView for the rebuild of the 1949 EDSAC computer at The National Museum of Computers at Bletchley Park, England.
Tony

**Comment by yytseng** [comment]
Hello, this program is developed by Thomas Weidenfeller @2008.

I just want to keep it available on internet.

for, MSO2204EA you may use another tool made by GWinstek.

https://github.com/OpenWave-GW/OpenWave-2KE

**Comment by tony1tf** [comment]
Thanks for your fast reply. The trouble with the OpenWave tools is they
don't have the protocol decoders that you can write for SigRok.
My volunteer job at the National Museum of Computing would be much more
difficult - debugging EDSAC as we build it - without the specialist tools.
I have looked at the GW Instek programming manuals and there is too much
for me to understand to be able to contribute to this software as I am
basically a hardware engineer.
Thanks anyway for keeping it alive here.
Tony
